shared/src/components/exercise-preview/controls-overlay.js

```diff
@@ -48,7 +48,7 @@ class ControlsOverlay extends React.Component {
                   className={`action ${type}`}
                   onClick={partial(this.onActionClick, partial.placeholder, action.handler)}
                 >
-                  <Icon type={iconTypes[type]} />
+                  {iconTypes[type] && <Icon type={iconTypes[type]} />}
                   <span>
```

Comment:
hah, this is good but we should also add a `details` icon above as well, I did that though so lets see if they conflict after this one is merged